packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -86,20 +86,20 @@ class SimpleFluidError implements IFluidErrorBase {
     readonly fluidErrorCode: string;
     readonly message: string;
     readonly stack?: string;
-    readonly name?: string;
+    readonly name: string = "Error";
     readonly errorInstanceId: string;
 
     constructor(
         errorProps: Omit<IFluidErrorBase,
-            "getTelemetryProperties" |
-            "addTelemetryProperties" |
-            "errorInstanceId">,
+            | "getTelemetryProperties"
+            | "addTelemetryProperties"
+            | "errorInstanceId"
+            | "name">,
     ) {
         this.errorType = errorProps.errorType;
         this.fluidErrorCode = errorProps.fluidErrorCode;
         this.message = errorProps.message;
         this.stack = errorProps.stack;
-        this.name = errorProps.name;
```

Comment:
Shouldn't it try to grab name as IFluidErrorBase passed in might have it?

Reply:
WF - This is only used when we have an unrecognized error, and we only pull message and stack those errors, so nothing to copy here.